work2/Negtive25/MyAnimalShop/com/petshop/Dog.java

```diff
@@ -0,0 +1,28 @@
+package com.petshop;
+
+public class Dog extends Animal {
+    boolean isVaccineInjected;
+
+    public Dog(String name, int age, String gender, double price, double weight, boolean isVaccineInjected) {
+        super(name, age, gender, price, weight);
+        species = "Dog";
+        this.isVaccineInjected = isVaccineInjected;
+    }
+    public String toString()
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Dog name: ").append(getName()).append(", Age: ").append(getAge()).append(", Gender: ").append(getGender())
+               .append(", Price: ").append(getPrice()).append(", Weight: ").append(getWeight()).append(", Vaccine Injected: ")
+               .append(getIfVaccineInjected());
+
+        return sb.toString();
+    }
+    String getIfVaccineInjected() {
```

Comment:
Why default decorate this Method?

Reply:
zhe name of getIfVaccineInjected, when it returns boolean, we name it as "isVaccineInjected", which is start with "is"